### PR TITLE
Zihan/wd 71 ionic signup

### PR DESCRIPTION
**Pull request by zihan0**
sign up page finished

### CODE REVIEW

File: UI/basicGUI/src/pages/signup/signup.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  var axiosConfig = {
+      headers: {
+          'Content-Type': 'application/json',
+          'accept' : '*/*',
+      }
+    };
+    if (this.passwd1==this.passwd2){
+      this.passwd = this.passwd1;
+      this.ax.post('http://localhost:8080/api/user/', {
+      'username': this.usrname,
```
Comment (bonellon) [inline UI/basicGUI/src/pages/signup/signup.ts:44]: usrname -> username

File: UI/basicGUI/src/pages/signup/signup.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  passwd1='';
+  passwd2='';
+  email='';
+  ax = require('axios');
+  constructor(public navCtrl: NavController, public navParams: NavParams,  public alertCtrl: AlertController) {
+  }
+  ionViewDidLoad() {
+    console.log('ionViewDidLoad SignupPage');
+  }
+  //alert for unmatch imput passwords
```
Comment (bonellon) [inline UI/basicGUI/src/pages/signup/signup.ts]: imput

File: UI/basicGUI/src/pages/signup/signup.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+          'Content-Type': 'application/json',
+          'accept' : '*/*',
+      }
+    };
+    if (this.passwd1==this.passwd2){
+      this.passwd = this.passwd1;
+      this.ax.post('http://localhost:8080/api/user/', {
+      'username': this.usrname,
+      'email': this.email,
+      'password': this.passwd
```
Comment (bonellon) [inline UI/basicGUI/src/pages/signup/signup.ts:46]: can we use 'password': this.passwd1 and avoid having 3 password variables?

**Review by bonellon — CHANGES_REQUESTED** [review]

File: UI/basicGUI/src/pages/signup/signup.ts (hunk 1 above)
Comment (zihan0) [reply UI/basicGUI/src/pages/signup/signup.ts:44]: no can do

File: UI/basicGUI/src/pages/signup/signup.ts (hunk 3 above)
Comment (zihan0) [reply UI/basicGUI/src/pages/signup/signup.ts:46]: done

File: UI/basicGUI/src/pages/signup/signup.ts (hunk 2 above)
Comment (zihan0) [reply UI/basicGUI/src/pages/signup/signup.ts]: fixed, thank you

**Review by bonellon — APPROVED** [review]

**Review by Jamesxu182 — APPROVED** [review]

**Pull request merged**